blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,286 @@
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h:mma Z
+common.date.readable.short=MMM DD h:mma Z
+#加载中...
+common.pager.loading=\u52a0\u8f7d\u4e2d\u002e\u002e\u002e
```

Comment:
I think you are right. Some translations are in unicode only. See line 238-245 in this file. These are the difficulties when reviewing.